### PR TITLE
Add option to display a Frame interactively

### LINKED ISSUE #1668: R Reticulate Support - Option to remove interactivity
I'm working with RStudio on an [issue](https://github.com/rstudio/reticulate/issues/431) involving the `datatable` Python API. We are using a library called `reticulate`, which does not currently support the interactivity shown in the image below.

image

Is it possible to have an option to turn off this interactivity so we can use via the `reticulate` within the RStudio IDE similar to how `pandas` is used?

### PR DESCRIPTION
**Pull request by st-pasha**
- Added option `dt.options.display.interactive` to control whether Frames will be displayed in interactive mode in a console or not.
- The default value of `dt.options.display.interactive` changed to `False`.
- Added a helper class `py::obool`.

Closes #1668

### CODE REVIEW

File: c/options.cc [hunk 1]
```diff
@@ -193,6 +197,9 @@ static py::oobj get_option(const py::PKArgs& args) {
   } else if (name == "frame.names_auto_prefix") {
     return py::ostring(frame_names_auto_prefix);
 
+  } else if (name == "display.interactive") {
+    return py::oint(display_interactive);
```
Comment (oleksiyskononenko) [inline c/options.cc]: Any reason there is no `py::obool`?

Comment (st-pasha) [reply c/options.cc]: i guess just because nobody added it

Comment (st-pasha) [reply c/options.cc]: fixed

Comment (oleksiyskononenko) [reply c/options.cc]: cool, that was fast :)

**Pull request merged**